packaging/debs/Debian/debian/control

```diff
@@ -7,11 +7,12 @@ Uploaders: Alvaro Videla <alvaro@rabbitmq.com>,
  Jean-Sébastien Pédron <jean-sebastien@rabbitmq.com>,
  Giuseppe Privitera <giuseppe@rabbitmq.com>
 Build-Depends: debhelper (>= 9),
+ dh-systemd (>= 1.5),
  erlang-dev,
  python-simplejson,
  xmlto,
  xsltproc,
- erlang-nox (>= 1:16.b.3),
+ erlang-nox (>= 1:16.b.3) | esl-erlang
```

Comment:
There is a comma missing at the end of this line. I fixed it and merged your branch.

Thank you very much!